calibration/FLORDG/CGINS-FLORDG-03286__20161010.csv

```diff
@@ -0,0 +1,10 @@
+serial,name,value,notes
+3286,CC_angular_resolution,1.096,Constant
+3286,CC_dark_counts_chlorophyll_a,0.081,in Volts (because analog signal going to CTDBP)
+3286,CC_dark_counts_volume_scatter,0.059,"in Volts (because analog signal going to CTDBP)
+GI01SUMO-RII11-02-FLORDG103 will be defined in Group 5"

```

Comment:
I don't think this line should be here.

Reply:
Yah - I think there are notes in a lot of these that are from the original files and were not deleted when new csvs were created from the originals.  Also - are we bothering with source_file info when touching files?  Thought we were - although if these are just date changes then probably not.

Reply:
This is okay.  It's just the notes line wrapping around.  We could have added source file.  Many of them are just file changes.  I have all of the files freshly available, so I can go in and add them. 
Should I remove the "GI01SUMO-RII11-02-FLORDG103 will be defined in Group 5" in the comments.

Reply:
Yeah that's definitely what happened. I think we should delete the line. I did this review before we were all committed to adding source file so I didn't add them here. I'll make sure to add them going forward to any file I touch, then hopefully we can automate the rest.